### DISCUSSION TITLE
Machine Learning Services

### DISCUSSION THREAD

**Discussion by tanjaamplo** [Amplo-GmbH/Platform-Documentation#4] [Ideas]
![image](https://user-images.githubusercontent.com/91560861/135417318-72cda118-fd83-4a5c-89e3-6712284ae338.png)
I was wondering if we should rename "Machine Learning Services" to "Smart Maintenance Services" to be consistent with the website etc. ?

**Comment by nielsuit227** [comment]
Mmm, with Condition Based Monitoring it's technically not machine learning. However, Smart Maintenance Services extend far beyond our data analyses. Not sure what's better suited. Your argument is solely less terminology?

**Reply by tanjaamplo** [reply]
Yep, I don't have a strong opinion :) I think we can keep it like this
